billing/checkout/service.go

```diff
@@ -407,7 +428,7 @@ func (s *Service) Create(ctx context.Context, ch Checkout) (Checkout, error) {
 			AutomaticTax: &stripe.CheckoutSessionAutomaticTaxParams{
 				Enabled: stripe.Bool(s.stripeAutoTax),
 			},
-			Currency: stripe.String(billingCustomer.Currency),
+			Currency: stripe.String(s.defaultCurrency),
```

Comment:
This makes billing customers' currency property obsolete and makes it a Frontier app property. Is it intended? 
If yes, we should consider removing this property from the Billing Customers.